Sources/SwiftSyntax/generated/raw/RawSyntaxValidation.swift

```diff
@@ -227,7 +227,12 @@ func validateLayout(layout: RawSyntaxBuffer, as kind: SyntaxKind) {
   case .accessPathComponent:
     assert(layout.count == 5)
     assertNoError(kind, 0, verify(layout[0], as: RawUnexpectedNodesSyntax?.self))
-    assertNoError(kind, 1, verify(layout[1], as: RawTokenSyntax.self, tokenChoices: [.tokenKind(.identifier)]))
```

Comment:
Seems like we didn't regenerate in https://github.com/apple/swift-syntax/commit/aea4442aba9f0393de87493b13df32962d95f37a?